hub/core/tensor.py

```diff
@@ -275,6 +276,30 @@ def append(self, sample: InputSample):
         """
         self.extend([sample])
 
+    def modified(self, target_id: Optional[str] = None, return_indexes: bool = False):
```

Comment:
I think the name should be more specific, like `get_modified_samples`.

Reply:
@istranic

Reply:
By default, it returns a tensor view. On passing return_indexes=True to it, you get a a tuple of tensor view and indexes.